StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml

```diff
@@ -108,77 +124,68 @@
     <element id="MedicationRequest.extension:responsiblePractitioner">
       <path value="MedicationRequest.extension" />
       <sliceName value="responsiblePractitioner" />
-      <short value="Responsible Practitioner for the prescription" />
-      <definition value="Must only be populated if the  `requester` can not be responsible for the prescription, i.e. they would not be named as the prescriber on the FP10.&#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA;&quot;extension&quot;: [&#xD;&#xA;        &#xD;&#xA;    {&#xD;&#xA;        &quot;url&quot;: &quot;https://fhir.nhs.uk/StructureDefinition/Extension-DM-ResponsiblePractitioner&quot;,&#xD;&#xA;        &quot;valueReference&quot;: {&#xD;&#xA;            &quot;reference&quot;: &quot;urn:uuid:a5acefc1-f8ca-4989-a5ac-34ae36741466&quot;,&#xD;&#xA;            &quot;display&quot;: &quot;DR SAZ RAZ&quot;&#xD;&#xA;        }&#xD;&#xA;    }&#xD;&#xA;],&#xD;&#xA;```" />
       <min value="0" />
       <max value="1" />
       <type>
         <code value="Extension" />
-        <profile value="https://fhir.nhs.uk/StructureDefinition/Extension-DM-ResponsiblePractitioner" />
+        <profile value="https://fhir.nhs.uk/StructureDefinition/Extension-England-DMResponsiblePractitioner" />
       </type>
-      <mustSupport value="true" />
+      <isModifier value="false" />
     </element>
     <element id="MedicationRequest.extension:prescriptionEndorsement">
       <path value="MedicationRequest.extension" />
       <sliceName value="prescriptionEndorsement" />
-      <short value="NHS BSA Prescription Endorsements" />
-      <definition value="The codes are contained in the valueset&#xD;&#xA;&#xD;&#xA;| Code | Display |&#xD;&#xA;|---|---|&#xD;&#xA;| CC |  Contraceptive |&#xD;&#xA;| FS |  Sexual Health| &#xD;&#xA;| ACBS |  Advisory Committee on Borderline Substances. Part XV Drug Tariff | &#xD;&#xA;| SLS |  Selected List Scheme. Part XVIIIB Drug Tariff | &#xD;&#xA;| AF |  Food replacement/food supplement products|&#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA;&quot;resourceType&quot;: &quot;MedicationRequest&quot;,&#xD;&#xA;    &quot;extension&quot;:  [&#xD;&#xA;      {&#xD;&#xA;          &quot;url&quot;: &quot;https://fhir.nhs.uk/StructureDefinition/Extension-DM-PrescriptionEndorsement&quot;,&#xD;&#xA;          &quot;valueCodeableConcept&quot;: {&#xD;&#xA;              &quot;coding&quot;:  [&#xD;&#xA;                {&#xD;&#xA;                    &quot;system&quot;: &quot;https://fhir.nhs.uk/CodeSystem/medicationrequest-endorsement&quot;,&#xD;&#xA;                    &quot;code&quot;: &quot;SLS&quot;,&#xD;&#xA;                    &quot;display&quot;: &quot;Selected List Scheme&quot;&#xD;&#xA;                }&#xD;&#xA;            ]&#xD;&#xA;        }&#xD;&#xA;    }&#xD;&#xA;  ],&#xD;&#xA;```" />
       <min value="0" />
+      <max value="*" />
```

Comment:
Like other instances, * added as max cardinality but not sure if this is right or not.